static/src/javascripts/projects/commercial/modules/article-body-adverts.ts

```diff
@@ -147,11 +152,22 @@ const addDesktopInlineAds = (isInline1: boolean): Promise<boolean> => {
 		filter: filterNearbyCandidates(adSizes.mpu.height),
 	};
 
+	let minAbove = 1000;
```

Comment:
What do you think about using a ternary here?

`let minAbove = isPaidContent ? 1600 : 1000`

Reply:
If you'd rather not, that's absolutely fine!

Reply:
I like the way you've done it! It was ok before when there was only one conditional. Now there's several I find it easier to parse without involving ternaries.

Reply:
Small thing but have you tested it on a paid content article?

Reply:
Yeah that's a good point. It's probably easier to add more conditions this way too.

Reply:
We have tested on a few. I couldn't find one where the Most Viewed island loads though

Reply:
Good point @simonbyford will check that!